templates/authors/author_index_page.html

```diff
@@ -0,0 +1,61 @@
+{% extends 'base_page.html' %}
+
+{% load static %}
+{% load wagtailcore_tags %}
+{% load wagtailimages_tags %}
+{% block breadcrumb %}{% endblock %}
+{% block content %}
+    {% include 'includes/generic-intro.html' with breadcrumb=True title=page.title intro=page.role %}
+
+    <div class="container mt-4" data-container-name="{{ page.title }}">
+        <h2>Discover all authors</h2>
+        <ul class="card-group--list-style-none">
+            {% for author_page in page.author_pages %}
+                
+                <li class="col-sm-12 col-md-6 col-lg-4 mb-3">
+                    <div class="card-group-secondary-nav">
+                        <a
+                            href="{{ author_page.url }}"
+                            class="card-group-secondary-nav__image-link"
+                            data-card-type="card-group-secondary-nav"
+                            data-link-type="Card image"
+                            data-card-title="{{ author_page.title }}"
```

Comment:
Do we use these `data-` attributes for tracking or something? I've seen them in quite a few places now.

Reply:
Yeah it's for tracking with GA4 😃 I've removed some of them from the bits I've copied from other elements just so I wasn't constantly changing the values of it 😅